xbmcaddon.py

```diff
@@ -220,15 +244,46 @@ def __get_strings(self):
 
         return translations
 
+    def __filter_lang_xmltag(self, tag, xml_content):
+        tag_matches = re.findall(r'<' + tag + r'(?:\s*lang=\"([a-zA-Z-]+)\")?\s*>(.*?)</' + tag + '>', xml_content, flags=re.DOTALL)
```

Comment:
Apparently the languages are en_GB, not en-GB (my bad there). The regex should have `lang=\"([a-zA-Z-_]+)\")?` instead of `lang=\"([a-zA-Z-]+)\")?`